### DISCUSSION TITLE
AI agents integration with Payload CMS.

### DISCUSSION THREAD

**Discussion by samiran-phukon** [payloadcms/payload#9833] [General]
I want to use AI agents to generate content using LangChain. Can it be integrated with Payload CMS? Can it be easily integrated with a custom function? If yes, how should I proceed? The generated content should be populated on the fields and shown to the content creator for review.

**Comment by DanRibbens** [comment]
Older thread, we did add cursor rules and agents.md to our templates recently. If your project is older and you haven't pulled those files in you should copy them to see if it improves your clanking.

https://github.com/payloadcms/payload/pull/14889

I can't speak to LangChain specifically but I'd be curious how it is going or if you have any tips so we can do to make it better.
